src/emcpy/plots/create_plots.py

```diff
@@ -242,6 +244,32 @@ def add_text(self, xloc, yloc, text, fontsize=12,
                      fontweight=fontweight, color=color,
                      alpha=alpha, ha=horizontalalignment)
 
+    def add_logo(self, xloc, yloc, zorder=10, which='noaa/nws',
+                 alpha=0.5):
+        """
+        Add NOAA/NWS logo. Requires user to adjust x and y locations.
+
+        Args:
+            xloc : (int/float) x location on figure in pixels
+            yloc : (int/float) y location on figure in pixels
+            zorder : (int; default=10) The z order of the logo
+            which : (str; default='noaa/nws') which type of logo
+                    to plot. Options include 'noaa', 'nws', or
+                    'noaa/nws'
+            alpha : (float; default=0.5) alpha of the image
+        """
+        image_dict = {
+            'noaa': 'noaa_logo_75x75.png',
+            'nws': 'nws_logo_75x75.png',
+            'noaa/nws': 'noaa_nws_logo_150x75.png'
+        }
+
+        image_path = emcpy.emcpy_directory + '/logos/' + image_dict[which]
```

Comment:
use `os.path.join` for Windows compatibility

Reply:
Where would you like this? The `emcpy.emcpy_directory` comes from the `__init__.py` file.

Reply:
```suggestion
        image_path = os.path.join(emcpy.emcpy_directory, 'logos' , image_dict[which])
```

Reply:
Ah got it, thank you.